strongbox-parent/pom.xml

```diff
@@ -62,7 +61,7 @@
         <version.spring.data>0.10.0-SNAPSHOT</version.spring.data>
         <version.orientdb>2.1.16</version.orientdb>
         <version.guava>19.0</version.guava>
-        <version.servlet.api>2.5</version.servlet.api>
+        <version.javaee>7.0</version.javaee>
```

Comment:
@AlexOreshkevich ,

I'm afraid I don't quite agree with this change. This means that you are trying the application to the J2EE bundle, which is massive, also it's not in line with JDK 1.8.

I think it's much better to use things the way they were previously -- in a modularized manned. Use only what's needed, not the whole bunch. We don't want a final `war` of half a gig (or more). Also, bundling the complete J2EE may introduce licensing issues and also break the container (as `war` files shouldn't really include the J2EE itself; any of it's modules have to be with `scope:provided`).